bot/__init__.py

```diff
@@ -168,6 +171,52 @@ def _get_word(self) -> str:
         result = self.buffer[self.previous:self.index + (pos+2)]
         self.index += 2
 
+    # Check if a command in the form of `bot.tags['ask']`
+    # or alternatively `bot.tags['ask'] = 'whatever'` was used.
+    elif current == "[":
+        def clean_argument(arg: str) -> str:
+            """Helper function to remove any characters we don't care about."""
+
+            return arg.strip("[]'\" ")
+
+        # Syntax is `bot.tags['ask']` => mimic `getattr`
+        if self.buffer.endswith("]"):
+            # Key: The first argument, specified `bot.tags[here]`
+            key = clean_argument(self.buffer[self.index:])
+            # note: if not key, this corresponds to an empty argument
+            #       so this should throw / return a SyntaxError ?
+            args = f"\"{key}\""
```

Comment:
The escape characters here can be avoided :eyes: 
`f"\"{key}\""` -> `f'"{key}"'`